packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -201,4 +201,160 @@ describe('babel plugin', () => {
 
     expect(actual).toInclude('c_MyDiv');
   });
+
+  it('should compress class name for styled component', () => {
+    const actual = transform(
+      `
+      import { styled } from '@compiled/react';
+
+      const MyDiv = styled.div\`
+        font-size: 12px;
+      \`;
+    `,
+      {
+        classNameCompressionMap: {
+          '1wyb1fwx': 'a',
+        },
+      }
+    );
+
+    expect(actual).toIncludeMultiple(['.a{font-size:12px}', 'ax(["_1wyb_a", __cmplp.className])']);
```

Comment:
Does the extracted CSS also contain
`._1wyb_a, .1wyb1fwx {font-size:12px}`?

Reply:
No, the extracted CSS looks like
```
.a {font-size:12px}
```